src/formatter-settings/types.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+export interface JavaFormatterSetting {
+    id: string;
+    name?: string;
+    value: string;
+    candidates?: string[];
+    catagory: Catagory;
+    exampleKind: ExampleKind;
+    valueKind?: ValueKind;
```

Comment:
When will valueKind be undefined?

Reply:
It seems `undefined` is meaningless for `valueKind`, will remove it.